module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -395,6 +395,28 @@ protected function getItemStatusGroup($record, $messages, $callnumberSetting)
         ];
     }
 
+    /**
+     * Support method for getItemStatuses() -- process a failed record.
+     *
+     * @param array $record Information on items linked to a single bib record
+     *
+     * @return array Summarized availability information
+     */
+    protected function getItemStatusError($record)
+    {
+        return [
+            'id' => $record[0]['id'],
+            'error' => $record[0]['error'],
```

Comment:
Should we be translating this here? Right now it's always in English regardless of the active language.

Reply:
Yeah, forgot that one.